lib/killrails.bash

```diff
@@ -0,0 +1,11 @@
+# Quick function to kill a daemonized Rails server
```

Comment:
This shouldn't be in the `lib` folder. Please move the file to the `plugins/available` folder and rename it to `killrails.plugin.bash` to follow the framework's naming conventions.